packages/react-dev-utils/WebpackDevServerUtils.js

```diff
@@ -113,12 +101,20 @@ function printInstructions(appName, urls, useYarn) {
   console.log();
 }
 
-function createCompiler(webpack, config, appName, urls, useYarn) {
+function createCompiler(
```

Comment:
Reminder for myself to update `README.md` of `react-dev-utils` before completing this PR